thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -291,26 +249,6 @@ export function toColor(urn) {
   return 'none';
 }
 
-export function toColorDirection(delta, inverse = false) {
```

Comment:
This is also being used by the tooltip builder, the color delta helper and 2 other components. Any reason why we're deleting? Let me know if I'm missing something cc @apucher @ttbach